routes/files.js

```diff
@@ -22,11 +48,18 @@ routes.push({
   options: {
     description: 'Get all files for an article',
     notes: 'Retrieve all files for a given article or page url.',
-    validate: {},
+    validate: {
+      params: {
+        articleUrl: Joi.string().uri(),
```

Comment:
Would it make sense to only allow `http` and `https` URLs?

```javascript
Joi.string().uri({ scheme: ['http', 'https'] })
```

— https://github.com/hapijs/joi/blob/master/API.md#stringurioptions

Reply:
Oh 🙈 I remember you already suggested that! Forgot about it, but will do, makes perfect sense here.

Reply:
This does somehow not appear in the `openapi.yml` though 🤔

Reply:
OpenAPI/Swagger doesn't support something like this as far as I know. You can add `.description('…')` to the Joi validator to provide a custom message and just mention that you expect a URL with these schemes. This'll show up in the OpenAPI docs.